INSTALL

```diff
@@ -640,12 +640,12 @@ The specific libraries and their options are:
   Options: OCCA_DIR, OCCA_OPT, OCCA_LIB.
   Versions: OCCA >= 1.0.9.
 
-- libCEED (optional), used when MFEM_USE_CEED = YES. Requires libCEED v0.6
+- libCEED (optional), used when MFEM_USE_CEED = YES. Requires libCEED v0.7
```

Comment:
TODO: Update git-hash upon v0.7 release.

Reply:
Actually, ideally this will just say libCEED v0.7. We should get rid of the hash if possible

Reply:
Updated.